lib/posix/spawn/child.rb

```diff
@@ -95,6 +95,31 @@ def initialize(*args)
           @options[:pgroup] = true
         end
         @options.delete(:chdir) if @options[:chdir].nil?
+
+        @out, @err = "", ""
```

Comment:
Might want to use `String.new` to avoid breaking when someone passes `--enable-frozen-string-literal`.